models/staging/tpdm/base/base_tpdm__evaluation_ratings.sql

```diff
@@ -0,0 +1,31 @@
+with evaluation_ratings as (
+    {{ source_edfi3('evaluation_ratings') }}
+),
+renamed as (
+    select
+        tenant_code,
+        api_year,
+        pull_timestamp,
+        last_modified_timestamp,
+        file_row_number,
+        filename,
+        is_deleted,
+
+        v:id::string                                  as record_guid,
+        v:sectionReference:sectionIdentifier::string  as section_id,
+        v:evaluationDate::date                        as evaluation_date,
+        v:comments::string                            as comments,
+        v:actualDuration::int                         as actual_duration,

```

Comment:
neither comments nor actual duration appear to exist in the source for this table